core/vibes/soul/sections/cart/client.tsx

```diff
@@ -390,13 +431,21 @@ export function CartClient<LineItem extends CartLineItem>({
               key={lineItem.id}
             >
               <div className="relative aspect-square w-full max-w-24 overflow-hidden rounded-xl bg-[var(--cart-image-background,hsl(var(--contrast-100)))] focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-[var(--cart-focus,hsl(var(--primary)))] focus-visible:ring-offset-4">
-                <Image
-                  alt={lineItem.image.alt}
-                  className="object-cover"
-                  fill
-                  sizes="(min-width: 28rem) 9rem, (min-width: 24rem) 6rem, 100vw"
-                  src={lineItem.image.src}
-                />
+                {lineItem.typename === 'CartGiftCertificate' ? (
```

Comment:
Gift Card certificate can't have images? 🤔 

An area of opportunity, I see gift card images all the time (not relevant right now).